cfpb/consumerfinance.gov
# Change flake8 source from gitlab to github | adjust formatting in config file

## Changes

- Change flake8 source from gitlab to github.
- Adjust formatting in precommit config file.


## How to test this PR

1. `pre-commit run --all-files` should not ask for a gitlab login. *NOTE:* it'll adjust migrations in the isort set. This can be ignored for now.
